tests/unit/test_current_user.py

```diff
@@ -0,0 +1,150 @@
+import pytest
+
+from mz_bokeh_package.utilities import CurrentUser, FetchUserInfoError
+from mz_bokeh_package.utilities import MZGraphQLClient
+
+API_KEY = "6fzQxEJL1YX9ldWUhHv5NPpiIcV7Gg8r03KwmeCqyRakj2nSAstDMTb4oB"
+SESSION_ID = "LME0EBfddYQCJqpZoO4ysYvynDXFLavC8hu3RzHW6e5e"
+USER_ID = "79e8e0f4-d2d6-4c16-b833-0a8986a6ce53"
+USER_NAME = "user_name"
```

Comment:
@ofir-frd - I agree with Stefan that we don't need to use strings of the same length, since we are not testing the formats of these strings. So for the security reasons Stefan mentioned, you can choose much shorter ones for the api_key, session id and user id.